flex/views/experiment/TabSampleSetupView.mxml

```diff
@@ -2319,10 +2234,12 @@
 			</mx:AddChild>
 		</mx:State>
 		<mx:State name="NanoStringState" basedOn="MDMiSeqState">
-			<mx:SetProperty target="{isPreppedContainer}" name="visible" value="false"/>
-			<mx:SetProperty target="{isPreppedContainer}" name="includeInLayout" value="false"/>
-			<mx:SetProperty target="{prePooledSamplesHBox}" name="includeInLayout" value="false"/>
-			<mx:SetProperty target="{prePooledSamplesHBox}" name="visible" value="false"/>
+
+			<mx:SetProperty target="{parentDocument.seqSetupView.isPreppedContainer}" name="visible" value="false"/>
+			<mx:SetProperty target="{parentDocument.seqSetupView.isPreppedContainer}" name="includeInLayout" value="false"/>
+			<mx:SetProperty target="{parentDocument.seqSetupView.prePooledSamplesHBox}" name="includeInLayout" value="false"/>
+			<mx:SetProperty target="{parentDocument.seqSetupView.prePooledSamplesHBox}" name="visible" value="false"/>
+
 		</mx:State>

```

Comment:
Same as above comment.  I believe this should be moved to the experiment edit view when it picks up on the state change.